paychmgr/paych.go

```diff
@@ -180,6 +180,20 @@ func (ca *channelAccessor) checkVoucherValidUnlocked(ctx context.Context, ch add
 		return nil, xerrors.Errorf("voucher ChannelAddr doesn't match channel address, got %s, expected %s", sv.ChannelAddr, ch)
 	}
 
+	// check voucher is unlocked
```

Comment:
There are two different senses of unlocked here. `checkVoucherValidUnlocked` means "check the voucher is valid without taking the lock because we're already holding it".